csharp/src/Ice/Communicator.cs

```diff
@@ -218,14 +223,18 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
             ILogger? logger = null,
             Instrumentation.ICommunicatorObserver? observer = null,
             TlsClientOptions? tlsClientOptions = null,
-            TlsServerOptions? tlsServerOptions = null)
+            TlsServerOptions? tlsServerOptions = null,
```

Comment:
Just like for Clone, we should sort these optional parameters in alphabetical order.

Reply:
Added https://github.com/zeroc-ice/ice/issues/997 to fix in a separate PR